src/Aspire.Hosting.Python/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+    <PropertyGroup>
+      <!-- This needs to be set to true before importing parent Directory.Build.props -->
+      <!-- In preview until the public API is validated. -->
+      <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
```

Comment:
Why isn't this in the csproj?

Reply:
See conversation here: https://github.com/dotnet/aspire/pull/6195#discussion_r1792442697

cc @joperezr

Reply:
Based on this PR this is the way to do it now
https://github.com/dotnet/aspire/pull/6195/files

Reply:
That is no longer the case, it was only for RC1 as we had things going RC1 and some other things staying in preview, but now everything should fall back to be preview or stable.